### DISCUSSION TITLE
Send-later may have an issue with virtual folders / saved searches

### DISCUSSION THREAD

**Discussion by Tom-2019** [Extended-Thunder/send-later#455] [Q&A]
I have been using send-later for a long time and every now and then I had a 'known' issue with the draft folders. I am using virtual folders or saved searches. When the send-later add-on is activated it sometimes claims that the draft folders cannot be found. When I deactivate the add-on the error does not occur.

**Comment by jikamens** [comment]
Is this still an issue with the current version of Thunderbird or Send Later?

**Comment by Tom-2019** [comment]
I have switched to Betterbird and they have fixed all issues with virtual folders completely. Since then I had no issues with send-later. You may close the issue from my side.
